contrib/stylesheets/_example.scss

```diff
@@ -179,11 +179,15 @@
   }
 
   .col-2-of-5 {
-    @include grid-column($columns: 2 of 5);
+    @media #{$small} {
+      @include grid-column($columns: 2 of 5);
+    }
   }
 
   .col-3-of-5 {
-    @include grid-column($columns: 3 of 5);
+    @media #{$small} {
+      @include grid-column($columns: 3 of 5);
+    }
```

Comment:
Are the above changes a direct impact of removing Bourbon? If not, can these go into another PR?